doc/build-osx.md

```diff
@@ -116,14 +116,7 @@ brew install berkeley-db@4
 ###### Descriptor Wallet Support
 
 Note: Apple has included a useable `sqlite` package since macOS 10.14.
-You may not need to install this package.
 
-`sqlite` is required to enable support for descriptor wallets.
-Skip if you don't intend to use descriptor wallets.
-
-``` bash
-brew install sqlite
-```
```

Comment:
Should we bring this in line with [the current description](https://github.com/bitcoin/bitcoin/blob/bb5bcf32baeb252d6f160f93b99d3e51715275c8/doc/build-osx.md?plain=1#L70-L75) instead?